src/main/kotlin/org/rust/ide/actions/RustGoToSuperModAction.kt

```diff
@@ -0,0 +1,21 @@
+package org.rust.ide.actions
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.CommonDataKeys
+import com.intellij.openapi.fileEditor.FileEditorManager
+import com.intellij.openapi.fileEditor.OpenFileDescriptor
+import com.intellij.openapi.project.DumbAwareAction
+import org.rust.lang.core.psi.impl.rustMod
+import org.rust.lang.core.psi.util.`super`
+
+class RustGoToSuperModAction : DumbAwareAction() {
+    override fun actionPerformed(e: AnActionEvent?) {
+        val project = e?.project ?: return
+        val psiFile = e?.getRequiredData(CommonDataKeys.PSI_FILE) ?: return
+        val superModFile = psiFile.rustMod?.`super`?.containingFile?.virtualFile ?: return
+
+        val descriptor = OpenFileDescriptor(project, superModFile, 0, 0)
+
+        FileEditorManager.getInstance(project).openTextEditor(descriptor, true) ?: return
```

Comment:
@matklad i think we'd better use `OpenFileDescriptor` here, `super.navigate` is really a concise way and it indeed works, but the problem is that it will not open the super mod file in the test fixture's in-memory editor, so it can not be tested, or maybe something i'm doing wrong here.